WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -2609,7 +2612,9 @@ e_moors = {
 	# Creation/Usurp Trigger
 	allow = {
 		OR = {
+			culture = maghreb_arabic
```

Comment:
The Maghrebs occupied the area of the Moors and northern Africa. Maghrebs are not your normal Arabs. They are part of that region.